### ISSUE TITLE
Q about database

### ISSUE THREAD

**Issue by OmurbekovaJyldyz** [idurar/idurar-erp-crm#1127]
dont know where should i past uri after creating cluster on MongoDB web? and should i do migration or smth like that?

**Comment by H9660** [comment]
Hi @OmurbekovaJyldyz . You can create a free MongoDB account and after creating a cluster copy the mongoDB uri. In the codebase search for .env file. There you would have an option to paste the database uri. Paste it over there and it's done. Make sure to restart the backend server.

**Comment by CodexRaunak** [comment]
That's correct and also don't forget to whitelist your IP address in the network. You can do it from Mongo Atlas dashboard in the Network Access.
